languages/csharp/concepts/attributes/links.json

```diff
@@ -1 +1,22 @@
-[]
+[
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/attributes/",
+    "description": "attribute concept"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/attributes",
+    "description": "attribute tutorial"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/attributes/#attribute-targets",
+    "description": "attribute target"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=net-5.0",
+    "description": "enum flags attribute"
+  },
```

Comment:
I think I would omit this from the list of links, as it is already linked to in the about.md and does IMHO not warrant a separate link.